department-of-veterans-affairs/vets-website
# Pension 69746 69748 69750 mvp step 3 health history

## Summary
On the Pension Benefits application(21P-527EZ),
Add Chapter 3 - Health and employment information

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Select 'Start your application without signing in'
3. Proceed to Chapter 3 by filling out the required fields on Chapter 1 and Chapter 2
4. Verify Chapter 3 matches acceptance criteria for associated tickets

## Related issue(s)
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/69746)
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/69748)
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/69750)
[Epic in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349)

## Testing done

- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots
_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

### Desktop
| Social Security Disability | Social Security Disability(No) | Medical Condition | Medical Condition(Yes) | Nursing Home | Required |
| ------ | ------ | ------ | ------ | ------ | ------ |
|![localhost_3001_pension_application_527EZ_medical_history_social-security-disability (1)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/17bf8aed-4c84-4d36-bc54-d16e8dd39cf0)|![localhost_3001_pension_application_527EZ_medical_history_social-security-disability](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/613d39c5-3816-487a-b6cd-2aea405eaa7e)|![localhost_3001_pension_application_527EZ_medical_history_social-security-disability (2)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/a0ff15be-44dd-4084-8dc5-432fc0113cee)|![localhost_3001_pension_application_527EZ_medical_history_social-security-disability (3)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/be526d83-1a16-4170-a597-8ace94e3c0a5)|![localhost_3001_pension_application_527EZ_medical_history_social-security-disability (4)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/218b792b-e0c4-41a2-a3bc-e0b0880ffa03)|![localhost_3001_pension_application_527EZ_medical_history_social-security-disability (5)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/af1d4b90-a983-42e7-a91b-eebfe2399482)





## What areas of the site does it impact?
Pension Benefits Application

## Acceptance criteria
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/69746)
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/69748)
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/69750)

### Quality Assurance & Testing

### Error Handling


### Authentication


### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:


## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
